modules/reference/pages/rpk/rpk-generate/rpk-generate-license.adoc

```diff
@@ -0,0 +1,52 @@
+= rpk generate license
+:description: rpk generate license
+
+Generate a trial license.
```

Comment:
```suggestion
Generates a trial license.
```
Quick glance at Google style guide recommends declarative instead of imperative (https://developers.google.com/style/reference-verbs); wasn't able to find the specific recommendation from Microsoft's guide. I think I'd prefer we move to Google's recommendation here and possibly add more specific rules to our Vale guide for reference-style docs for cases it doesn't already cover.